### DISCUSSION TITLE
SPFx ACE not showing up on dashboard on mobile

### DISCUSSION THREAD

**Discussion by sairao77** [SharePoint/sp-dev-docs#8002] [Q&A] [answered]
Hi,

Is there a specific attribute that needs to be changed in order for the custom AC developed using SPFx to be shown on Teams mobile viva dashboard?

**Comment by sairao77** [comment, marked as answer]
Found the answer, didn't package it as a production package.

**Comment by AJIXuMuK** [comment]
@sairao77 - could you provide few details here?
Are you able to add the ACE to the dashboard?
Is it visible on the dashboard in web browser or Viva Home?

**Reply by sairao77** [reply]
@AJIXuMuK - Sure!
I was able to add ACE to the dashboard on SP to begin with and no issues, however it was not showing up on the Viva dashboard on the mobile app. When I packaged the ACE --ship and updated the SP catalog with the new package (.sppkg), it started to show on the Viva dashboard on the mobile.

**Reply by AJIXuMuK** [reply]
Oh, got it! so PROD does work and DEV doesn't.
I thought vice-versa and it looked weird to me.
Thanks for confirming!
